### PR TITLE
Add bnb number of parameters test

### PR DESCRIPTION
**Pull request by carmocca**

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -135,3 +137,23 @@ def test_num_parameters():
     assert num_parameters(model) == 6
     assert num_parameters(model, requires_grad=True) == 4
     assert num_parameters(model, requires_grad=False) == 2
+
+
+@RunIf(min_cuda_gpus=1)
+@pytest.mark.parametrize("mode", ["nf4", "nf4-dq", "fp4", "fp4-dq", "int8", "int8-training"])
+@pytest.mark.skip("To be fixed")
```
Comment (carmocca) [inline tests/test_utils.py:144]: Here's your homework @Andrei-Aksionov :books:

File: tests/test_adapter.py [hunk 2]
```diff
@@ -4,9 +4,10 @@
 from unittest.mock import Mock
 
 import torch
-from conftest import RunIf
 from lightning import Fabric
 
+from conftest import RunIf
```
Comment (Andrei-Aksionov) [inline tests/test_adapter.py:9]: And then all these import reorders will be undone with automatic code fixes 😄.

File: tests/test_utils.py (hunk 1 above)
Comment (Andrei-Aksionov) [reply tests/test_utils.py:144]: Thanks a lot, of course 😆.
But, teacher, I don't understand the task.

That problematic PR #715 updated the way to count number of parameters (to reflect changes in BNB 0.41.2), but introduced another problem #734.
So this test should pass without any problems.

File: tests/test_adapter.py (hunk 2 above)
Comment (carmocca) [reply tests/test_adapter.py:9]: Uh maybe I ran ruff incorrectly? It's fine. Let the linters do their thing

**Pull request merged**